be/src/exprs/runtime_filter.h

```diff
@@ -276,20 +273,21 @@ class IRuntimeFilter {
 
     Status merge_from(const RuntimePredicateWrapper* wrapper);
 
-    // for ut
     static Status create_wrapper(RuntimeFilterParamsContext* state,
                                  const MergeRuntimeFilterParams* param, ObjectPool* pool,
                                  std::unique_ptr<RuntimePredicateWrapper>* wrapper);
     static Status create_wrapper(RuntimeFilterParamsContext* state,
                                  const UpdateRuntimeFilterParams* param, ObjectPool* pool,
                                  std::unique_ptr<RuntimePredicateWrapper>* wrapper);
+
     static Status create_wrapper(RuntimeFilterParamsContext* state,
-                                 const UpdateRuntimeFilterParamsV2* param, ObjectPool* pool,
-                                 std::unique_ptr<RuntimePredicateWrapper>* wrapper);
+                                 const UpdateRuntimeFilterParamsV2* param,
```

Comment:
The lifecycle of wrapper is not consistency. Sometimes it is in pool, but sometimes we use unique ptr

Reply:
This lifecycle is more complex in RF, the use of Pool is to better reuse memory, and the next PR will refactor this part of the code and unify it into Pool.
after update the patch，next version of doris only use V2， we can delete the old `update filter` api